Backends/PostScript/font/font.lisp

```diff
@@ -300,3 +299,12 @@
       (define-font-metrics dict-name ascent descent angle char-infos font-name))
     (setf (text-style-mapping port text-style) font-name)
     text-style))
+
+(defmethod climb:font-character-width :around ((font postscript-font-name) character)
```

Comment:
This probably doesn't need to be an `:around` method.

Reply:
you are right, follo a force push